src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesCloud.java

```diff
@@ -98,6 +98,7 @@
     @Nonnull
     private List<PodTemplate> templates = new ArrayList<>();
     private String serverUrl;
+    private String https_proxy;
```

Comment:
Follow java conventions (same for method names, parameters, etc.)

```suggestion
    private String httpsProxy;
```

Reply:
Makes sense. I'll make the edit.